src/libraries/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/ExternDll.cs

```diff
@@ -6,12 +6,12 @@ namespace System
 {
     internal static class ExternDll
     {
-#if FEATURE_PAL && !SILVERLIGHT
+#if TARGET_UNIX && !SILVERLIGHT
```

Comment:
Everything under this part of the ifdef can be deleted - keep just the else part.

Reply:
@jkotas `SILVERLIGHT` is not defined.  This is the currently active code.  I assume you want me to keep the active code and delete the `SILVERLIGHT` reference

Reply:
This .dll is not building for Unix, and I do not think it will ever change.

Reply:
I see what you are saying.  Everything before the `else` looks bad.